src/components/TooltipGlobal/TooltipGlobal.integration.test.js

```diff
@@ -0,0 +1,84 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { MockJetsContextProvider } from '../../__mocks__/MockJetsContext';
+
+import { JetsTooltipGlobal } from './index';
+import { activeTooltipData, paramsData } from './__fixtures__';
+
+const setup = ({ data = {}, config = {}, params = {}, events = {} } = {}) => ({
+  user: userEvent.setup(),
+  ...render(
+    <MockJetsContextProvider config={config} params={{ ...paramsData(params) }} events={events}>
+      <JetsTooltipGlobal data={{ ...activeTooltipData(data) }} />
+    </MockJetsContextProvider>
+  ),
+});
+
+describe('JetsTooltipGlobal', () => {
+  describe('when a button in the tooltip is pressed', () => {
+    it('should fire onSeatSelect when "Select" is pressed', async () => {
+      const onSeatSelect = jest.fn();
+      const { user } = setup({ events: { onSeatSelect } });
+
+      // need to remove mocked functions from the expected object
+      const expectedData = { ...activeTooltipData() };
+      delete expectedData.seatmapElement;
+      delete expectedData.seatNode;
+
+      await user.click(screen.getByText(/Select/));
+      expect(onSeatSelect).toHaveBeenCalledTimes(1);
+      expect(onSeatSelect).toHaveBeenCalledWith(expect.objectContaining(expectedData));
+    });
+
+    it('should not fire onSeatSelect when seat selection is disabled and "Select" is pressed', async () => {
+      const onSeatSelect = jest.fn();
+      const { user } = setup({ events: { isSeatSelectDisabled: () => true, onSeatSelect } });
+
+      await user.click(screen.getByText(/Select/));
+      expect(onSeatSelect).not.toHaveBeenCalled();
+    });
+
+    it('should fire onSeatUnselect when "Unselect" is pressed', async () => {
+      const onSeatUnselect = jest.fn();
+      const { user } = setup({ data: { passenger: { passengerLabel: 'Dave Smith' } }, events: { onSeatUnselect } });
+
+      // need to remove mocked functions from the expected object
+      const expectedData = { ...activeTooltipData() };
```

Comment:
This is interesting, so all the data is passed then to the select and unselect events? I thought that it would be just the seat-related information